lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -37,4 +37,5 @@
   font-size: $nx-font-size-xs;
   line-height: 16px;
   margin-top: $nx-spacing-xs;
+  position: absolute;
```

Comment:
Since deliberately overflowing the box is something we very rarely do, I think this deserves a comment about why

Reply:
> Since deliberately overflowing the box is something we very rarely do, I think this deserves a comment about why

https://github.com/sonatype/sonatype-react-shared-components/pull/180/commits/ecc34f01f80978a2473b5bf995583c97a0dc89f7